app/controllers/events/attendances_controller.rb

```diff
@@ -1,18 +1,28 @@
 # frozen_string_literal: true
 
 class Events::AttendancesController < ApplicationController
+  before_action :set_event, only: %i[create destroy]
+
   def create
-    @event = Event.find(params[:event_id])
     event_attendance = current_user.attend(@event)
-    (@event.attendees - [current_user] + [@event.user]).uniq.each do |user|
-      NotificationFacade.attended_to_event(event_attendance, user)
+    if event_attendance.id
```

Comment:
```suggestion
    if event_attendance.present?
```
でしょうか。 現状でもtrueになる場合もあるかと思いますが、 falseの場合 nil に対して .id となるためエラーになりそうですね。

Reply:
8a0362964b6468c78e43b8173427f01575ed8bd0
にて修正いたしました。
「event_attendance.present?」がtrueの場合もあると思いますので、「event_attendance.present?」を見てから「event_attendance.id」もチェックするように修正いたしました。